drivers/cst816s/include/cst816s_internal.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef CST816S_INTERNAL_H
+#define CST816S_INTERNAL_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
```suggestion

/* @name    Constants
 * @{
 */
```

/* @name    Registers
 * @{
 */

Reply:
Added